aiosmtpd/smtp.py

```diff
@@ -207,6 +263,35 @@ def smtp_QUIT(self, arg):
             self.transport.close()
 
     @asyncio.coroutine
+    def smtp_STARTTLS(self, arg):
+        log.info('===> STARTTLS')
+        if arg:
+            yield from self.push('501 Syntax: STARTTLS')
+            return
+        if (not self.tls_context) or (not _has_ssl):
+            yield from self.push(
+                '454 TLS not available due to temporary reason'
```

Comment:
Why the temporary failure here and the message?  Is it expected that if the client tries again in the future, `STARTTLS` will succeed?  I think it would be good to document the reasoning here in a comment.